code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -159,6 +161,7 @@
 	if(lying_angle || resting)
 		plasma_gain *= 2  // Doubled for resting
 
+	plasma_gain *= weeds.color_variant == RESTING_COLOR ? 1.2 : 1
```

Comment:
Why is this not just a weeds var

Reply:
I don't see why it should

Reply:
Because it means that A) the behavior is generic B) no magic numbers